### ISSUE TITLE
SideNav items not visible when focused

### ISSUE THREAD

**Issue by davidicus** [carbon-design-system/carbon-components-react#2151]
## Detailed description

> Describe in detail the issue you're having.

When using keyboard navigation you can open and focus items that are not visible in the sidenav. This also causes a collapse of its parent icon in the collapsed SideNav

> Is this issue related to a specific component?
SideNav component

> What did you expect to happen? What happened instead? What would you like to see changed?

Expected focused item to be visible

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?
Latest

## Steps to reproduce the issue

1. Collapse SideNav
2. Tab through until you are on a menu item with subnav
3. open and tab in
4. see collapse

## Additional information

- I have submitted [this](https://github.com/IBM/carbon-components/pull/2261) and [this](https://github.com/IBM/carbon-components-react/pull/2150) PR to resolve issue. 

![image](https://user-images.githubusercontent.com/1590966/55510734-e1e5e300-562c-11e9-8d61-c1ded7c3e099.png)

**Comment by asudoh** [comment]
Closing with an assumption that #2150 addresses this issue. Can re-open if anybody sees otherwise.
